clients/cmd/logstash/README.md

```diff
@@ -1,4 +1,21 @@
-# Contributing to Loki Logstash Output Plugin
+# Fork of Loki Logstash Output Plugin
```

Comment:
I think the doc was meant for a fork so you should adjust this please.

Reply:
Please add a separate section with a description of the new plugin features